src/material/chips/chip-list.ts

```diff
@@ -154,7 +154,7 @@ export class MatChipList extends _MatChipListBase implements MatFormFieldControl
 
   /** The array of selected chips inside chip list. */
   get selected(): MatChip[] | MatChip {
-    return this.multiple ? this._selectionModel.selected : this._selectionModel.selected[0];
+    return this.multiple ? this._selectionModel?.selected : this._selectionModel?.selected[0];
```

Comment:
The `MatChip[] | MatChip` type here seems off, since this can return undefined. I don't think we can fix it without a breaking change, but in the multiple case we could at least return an empty array?

Reply:
Changed, although my thinking is that this is somewhat of an edge case. You'll have to try access `selected` before change detection has run in order for it to break (e.g. by doing it in the constructor). I submitted the fix since we had a similar issue report in `mat-select`.